### ISSUE TITLE
Option to not insert the iframe into HTML until the user clicks Load

### ISSUE THREAD

**Issue by ruvilonix** [StarCitizenWiki/mediawiki-extensions-EmbedVideo#44]
I would like EmbedVideo to have the option to not preload the iframe html until the user clicks Load on that video, in order to reduce load on the browser when there are many videos on a page.

## Detailed Description

It would be great if the parser function had an option to not have the iframe at all (or any HTML except the div and a load button), and when the user clicks Load, then the div is populated with all the appropriate HTML to play the video. That way, the browser doesn't have to deal with 200 iframes every time the user opens the page, which slows everything way down. 

## Context

This would allow for much for flexibility in the use of this extension for people who want many videos per page.

I have a page with over 200 videos, and it will probably increase even more. I also have javascript that styles those videos using the DataTables library. The javascript can not run until all of those video iframes are processed, and with so many videos, it can take up to 20 seconds, or much more on mobile, for the javascript to begin. And my computer fan goes full blast every time I load the page.

I was able to speed the page up a lot to around 2-3 seconds by adding this CSS, but it's still slow compared to an identical page with no videos: 
```
.embedvideo iframe {display:none;}
.dataTables_scrollBody .embedvideo iframe { display: block }
```

## Possible Implementation

Maybe the function could have a new option for this: `{{#ev:service|id|delay=true}}`

**Comment by alistair3149** [comment]
Perhaps we should avoid generating the consent overlay in PHP, instead we should add it in JavaScript as it doesn't work without JS anyways.

Relevant discussion here: https://discord.com/channels/178359708581625856/1072276738719105054

**Comment by octfx** [comment]
Implemented in v3.2.4
